chat/models.py

```diff
@@ -21,3 +21,11 @@ class Message(models.Model):
     deleted = models.BooleanField(default=False)
     tokens = models.IntegerField(default=0)
     created_at = models.DateTimeField(auto_now_add=True)
+
+
+class SQLQuery(models.Model):
+    chat = models.ForeignKey(Chat, on_delete=models.CASCADE, related_name='sql_queries')
+    question = models.TextField()
+    sql_query = models.TextField()
+    results = models.JSONField()
```

Comment:
Not sure if we really wanna keep the results in the db... that could end up being very large.
Maybe just some meta of the query instead, result size, execution time...